reference/v8.json

```diff
@@ -394,7 +394,7 @@
       "type": "boolean",
       "function": "piecewise-constant",
       "default": false,
-      "doc": "If true, the icon will be visible even if it collides with other icons and text.",
+      "doc": "If true, the icon will be visible even if it collides with other previously drawn symbols.",
```

Comment:
This can be more explicit and say "collides with symbols from layers which appear later in the style."